include/agl/aglGPUMemBlock.hpp

```diff
@@ -27,6 +28,9 @@ class GPUMemBlockBase {
     bool tryAllocBuffer_(u64, sead::Heap*, s32, MemoryAttribute);
     void setBuffer_(u64, void*, void*, MemoryAttribute);
     void setVirtual_(u64, sead::Heap*, MemoryAttribute, GPUMemVoidAddr, s32);
+    void initializeGfxMemoryPool(
```

Comment:
If you pull in gfx_Types instead of gfx_Common you can just do ``nn::gfx::MemoryPool*`` here